docs/docs/native-docker.md

```diff
@@ -44,7 +44,8 @@ Docker version 1.0.0 or later installed on each slave node.
 
 ### Configure marathon
 
-1. Increase the marathon [command line option]({{ site.baseurl }}/docs/command-line-flags.html") `--task_launch_timeout` to at least the executor timeout you set on your slaves in the previous step.
+1. Increase the marathon [command line option]({{ site.baseurl }}/docs/command-line-flags.html)
+`--task_launch_timeout` to at least the executor timeout you set on your slaves in the previous step.
```

Comment:
We just merged a PR  (#1079) that increases the timeout to 5 minutes.  It might also be worth noting that the time unit for this option is `ms`, so `300000` means five minutes.